jetty-core/jetty-server/src/main/java/org/eclipse/jetty/server/handler/DelayedHandler.java

```diff
@@ -17,14 +17,14 @@
 import java.nio.charset.StandardCharsets;
 import java.util.Objects;
 import java.util.concurrent.CompletableFuture;
-import java.util.concurrent.atomic.AtomicReference;
 
 import org.eclipse.jetty.http.HttpField;
 import org.eclipse.jetty.http.HttpHeader;
 import org.eclipse.jetty.http.HttpHeaderValue;
 import org.eclipse.jetty.http.HttpStatus;
 import org.eclipse.jetty.http.MimeTypes;
-import org.eclipse.jetty.io.Content;
+import org.eclipse.jetty.http.MultiPartConfig;
+import org.eclipse.jetty.http.MultiPartFormData;
```

Comment:
If we restore delayed dispatch, I am not sure this Handler is necessary anymore.

My point being that we have one-liner helpers that do exactly the same without the need to have a Handler in the Handler chain.

For example, an application can call `FormFields.from()` as a one liner to do the same job.
We offer no synchronous API to get the `Fields`, so an application would always have to deal with a `CompletableFuture<Fields>`, so the `DelayedHandler` seems not that useful anymore.